docs/website/content/docs/bootstrapping.md

```diff
@@ -1,28 +1,129 @@
+
+```bash
+$ show service dhcp-server shared-network-name MetalDHCP
+
+ authoritative enable
+ subnet 192.168.254.0/24 {
+     default-router 192.168.254.1
+     dns-server 192.168.1.200
+     lease 86400
+     start 192.168.254.2 {
+         stop 192.168.254.252
+     }
+     subnet-parameters "include &quot;/etc/dhcp/ipxe-metal.conf&quot;;"
+ }
+```
+
+And this is my ipxe-metal.conf file.
+Notice that it sets a static address for the management node that I'll be booting, in addition to providing the "next server" info.
+This "next server" IP address will match references to `PUBLIC_IP` found below in this guide.
+
+```bash
+$ cat /etc/dhcp/ipxe-metal.conf
+
+allow bootp;
+allow booting;
+
+group "management-plane" {
+  next-server 192.168.1.150;
+  if exists user-class and option user-class = "iPXE" {
+    filename "http://192.168.1.150:8081/boot.ipxe";
+  } else {
+    filename "ipxe.efi";
+  }
+
+  host talos-mgmt-0 {
+     fixed-address 192.168.254.2;
+     hardware ethernet d0:50:99:d3:33:60;
+  }
+}
+```
+
 ## Create a Local Cluster
 
+The `talosctl` CLI tool has built-in support for spinning up Talos in docker containers.
+Let's use this to our advantage as an easy Kubernetes cluster to start from.
+
+Set an environment variable called `PUBLIC_IP` which is the "public" IP of your machine.
+Note that "public" is a bit of a misnomer.
+We're really looking for the IP of your machine, not the IP of the node on the docker bridge (ex: `192.168.1.150`).
+
+```bash
+export PUBLIC_IP="192.168.1.150"
+```
+
+We can now create our Docker cluster.
+Issue the following to create a single-node cluster:
+
 ```bash
 talosctl cluster create \
   -p 69:69/udp,8081:8081/tcp,9091:9091/tcp,50100:50100/tcp \
   --workers 0 \
   --endpoint $PUBLIC_IP
 ```
 
-## Untaint the Control Plane Node
+Note that there are several ports mentioned in the command above.
+These allow use to access the services that will get deployed on this node.
+
+Once the cluster create command is complete, issue `talosctl kubeconfig /desired/path` to fetch the kubeconfig for this cluster.
+You should then set your `KUBECONFIG` environment variable to the path of this file.
+
+## Untaint Control Plane
+
+Because this is a single node cluster, we need to remove the "NoSchedule" taint on the node to make sure non-controlplane components can be scheduled.
 
 ```bash
 kubectl taint node talos-default-master-1 node-role.kubernetes.io/master:NoSchedule-
 ```
 
-## Patch the Metadata Server
+## Install Sidero
+
+As of Cluster API version 0.3.9, Sidero is included as a default infrastructure provider in clusterctl.
+
+To install Sidero and the other Talos providers, simply issue:
+
+```bash
+clusterctl init -b talos -c talos -i sidero
+```
+
+## Patch Components
+
+We will now want to ensure that the Sidero services that got created are publicly accessible across our subnet. This will allow the metal machines to speak to these services later.
```

Comment:
We don't have one-sentence-per-line linter yet, but it would have hit you here :smile: 
```suggestion
We will now want to ensure that the Sidero services that got created are publicly accessible across our subnet.
This will allow the metal machines to speak to these services later.
```

Reply:
dang, thought I was doing so good on one sentence per line lol

Reply:
Lol yeah I saw that :smile: